robocopkaka/tafrica-copy
# Ft setup user authentication

#### What does this PR do?
Adds a home page and sets up user authentication
#### Description of Task to be completed?
* Add a home page
* Setup devise
* Add styles for signup page
* Add styles for login page
* Add styles for forgot password page
#### How should this be manually tested?
* Start the application
* Go to the home page
* You should see a background image and a button to login or signup at the top
#### Any background context you want to provide?
N/A
